### PR TITLE
[Novel Addition] Updated PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by kubawolanin**
Added `TESTING` section.

cc: @ThomDietrich @martinvw 

Signed-off-by: Kuba Wolanin <hi@kubawolanin.com> (github: kubawolanin)

### CODE REVIEW

**Comment by ThomDietrich** [comment]
Excellent addition! Did you want to capitalize "Maintainers"? I wonder if all blank lines are needed.

One general comment: @kubawolanin after the initial addition of the two templates I have the feeling, that the comment markers per line lessen readability. How would the two of you feel about adding just a starting and ending marker for the whole template or for paragraphs?

**Comment by kubawolanin** [comment]
Hey @ThomDietrich thanks for the review!
I completely agree with you regarding readability. I've removed the brackets.
Thanks!

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,38 +1,56 @@
...
+- Does your contribution follow the coding guidelines:
+  http://docs.openhab.org/developers/development/guidelines.html
+- Did you check for any (relevant) issues from the static code analysis:
+  http://docs.openhab.org/developers/development/bindings.html#static-code-analysis
+- Did you sign-off your work:
+  http://docs.openhab.org/developers/contributing/contributing#sign-your-work
+
+# TESTING
+
+Your Pull Request will be automatically built and available in the following folder:
```
Comment (ThomDietrich) [inline PULL_REQUEST_TEMPLATE.md]: "under the following URL" ?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,38 +1,56 @@
...
+https://openhab.jfrog.io/openhab/libs-pullrequest-local/org/openhab/
+
+It's a good practice to add an URL to your built JAR in this Pull Request description,
+so it's easier for the community to test your Add-on.
+If your Pull Request contains a new binding, it will likely wait some time
```
Comment (ThomDietrich) [inline PULL_REQUEST_TEMPLATE.md]: *take?

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,38 +1,56 @@
...
+before it's reviewed and processed by maintainers.
+That said, consider submitting your Add-on in the Eclipse IoT Marketplace
+See this thread for more info:
+https://community.openhab.org/t/distributing-bindings-through-the-iot-marketplace/24491
```
Comment (ThomDietrich) [inline PULL_REQUEST_TEMPLATE.md]: Could also be reduced to https://community.openhab.org/t/24491

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,38 +1,56 @@
...
+
+Don't forget to submit a thread about your Add-on in the openHAB Community:
```
Comment (ThomDietrich) [inline PULL_REQUEST_TEMPLATE.md]: community in lower case?

File: PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,38 +1,56 @@
...
-<!-- - Did you describe new features for the end user? -->
-<!-- - Did you describe any noteworthy changes in usage for the end user? --> 
-<!-- - Was the documentation updated accordingly, e.g. the addon README? -->
-<!-- - Does your contribution follow the coding guidelines: -->
-<!--   http://docs.openhab.org/developers/development/guidelines.html -->
-<!-- - Did you check for any (relevant) issues from the static code analysis: -->
-<!--   http://docs.openhab.org/developers/development/bindings.html#static-code-analysis -->
-<!-- - Did you sign-off your work: -->
-<!--   http://docs.openhab.org/developers/contributing/contributing#sign-your-work -->
+<!--
```
Comment (martinvw) [inline PULL_REQUEST_TEMPLATE.md]: The benefit of the old style (with the begin/end of line comments) was that you could interleave your text with the comments for a better overview, if you do that now, your text will become invisible. However I think it is more important for the issue template, we use real headers interleaved with the comments there.

File: PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,38 +1,56 @@
...
+- Does your contribution follow the coding guidelines:
+  http://docs.openhab.org/developers/development/guidelines.html
+- Did you check for any (relevant) issues from the static code analysis:
+  http://docs.openhab.org/developers/development/bindings.html#static-code-analysis
+- Did you sign-off your work:
+  http://docs.openhab.org/developers/contributing/contributing#sign-your-work
+
+# TESTING
+
+Your Pull Request will be automatically built and available under the following folder:
```
Comment (martinvw) [inline PULL_REQUEST_TEMPLATE.md]: 'will be automatically built' or 'will automatically be built'

**Comment by ThomDietrich** [comment]
Will you also add the comments marker change to the issues template?

Looks good! ;)

**Comment by martinvw** [comment]
> Will you also add the comments marker change to the issues template?

No, do not make it one big comment there because we have real titles there which end up in the message. There is a better use case there for having them separate so you can interleave your message with the hints/guidelines.

**Comment by kubawolanin** [comment]
@martinvw I've reverted the comment block change and applied changes you suggested.
Thanks!

**Comment by ThomDietrich** [comment]
@kubawolanin I think this was aimed at the issues template.
I would suggest to go the middle path and to put sections into single comments blocks. That would improve the readability while still making the headlines visible.

**Comment by martinvw** [comment]
> @kubawolanin I think this was aimed at the issues template.

Indeed :-)

**Comment by ThomDietrich** [comment]
@martinvw your opinion on the "middle path"?

**Comment by martinvw** [comment]
I don't like the "middle path" there is only one way 😉 

More serious, yes that makes sense 👍

**Comment by kubawolanin** [comment]
Sorry guys I don't understand what exactly you expect me to do :wink: 
You want me to keep the comment-lines or revert the single comment block back?
I didn't touch issue template at all - it can be adjusted further ;-)

**Comment by ThomDietrich** [comment]
Hey @kubawolanin, sorry for the confusion I've created. My suggestion is to follow this style:

```
<!--
Thanks for contributing to the openHAB project!
Please describe the goal and effect of your PR here.
Pay attention to the below notes and to *the guidelines* for this repository.
Feel free to delete any comment lines in the template (starting with "<!--").
-->

<!-- TITLE -->

<!-- 
...
```

**Comment by kubawolanin** [comment]
Alrighty! Thank you guys for your suggestions! 🍷

**Comment by ThomDietrich** [comment]
@kubawolanin there is one sentence actually related to comments and how these parts can be removed. I fear you'll need to update it...

**Comment by martinvw** [comment]
Thank you very much!

**Pull request merged**